src/Symfony/OtelSdkBundle/DependencyInjection/Configuration/Closure/MissingAttributesCallback.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Symfony\OtelSdkBundle\DependencyInjection\Configuration\Closure;
+
+use Closure;
+use OpenTelemetry\Symfony\OtelSdkBundle\DependencyInjection\ConfigurationException;
+
+class MissingAttributesCallback
+{
+    public static function createCheck(array $requiredAttributes): Closure
+    {
+        return Closure::fromCallable(static function (?array $config = []) use ($requiredAttributes) : bool {
+            $config = $config ?? [];
+            foreach ($requiredAttributes as $attr) {
+                if (!isset($config[$attr])) {
+                    return true;
+                }
+            }
+
+            return false;
+        });
+    }
+
+    public static function createExceptionTrigger(array $requiredAttributes): Closure
```

Comment:
These two still need to be separated into dedicated Closure factories with a simple *create* methods as in the other one.